### PR TITLE
cli: Add `json` and `-t` flags to `server members` command

### PR DESCRIPTION
**Pull request by Juanadelacuesta**

### CODE REVIEW

File: website/content/docs/commands/server/members.mdx [hunk 1]
```diff
@@ -57,3 +61,40 @@ server-1.global  10.0.0.8   4648  alive   true    2         3             1.3.0
...
+      },
+      "Status": "alive",
+      "ProtocolMin": 1,
+      "ProtocolMax": 5,
+      "ProtocolCur": 2,
+      "DelegateMin": 2,
+      "DelegateMax": 4,
+      "DelegateCur": 4
+    }
+  ]
```
Comment (jrasell) [inline website/content/docs/commands/server/members.mdx:92]: suggestion to use slightly more recent output as it has additional information populated:
```suggestion
[
    {
        "Addr": "127.0.0.1",
        "DelegateCur": 4,
        "DelegateMax": 5,
        "DelegateMin": 2,
        "Name": "bacon-mac.global",
        "Port": 4648,
        "ProtocolCur": 2,
        "ProtocolMax": 5,
        "ProtocolMin": 1,
        "Status": "alive",
        "Tags": {
            "build": "1.5.0",
            "region": "global",
            "rpc_addr": "127.0.0.1",
            "role": "nomad",
            "raft_vsn": "3",
            "bootstrap": "1",
            "expect": "1",
            "dc": "dc1",
            "port": "4647",
            "revision": "fc40c491cacec3d8ec3f2f98cd82b9068a50797c",
            "vsn": "1",
            "id": "354fd9a8-4228-ca8e-8b93-3cf002514910"
        }
    }
]
```

File: website/content/docs/commands/server/members.mdx [hunk 2]
```diff
@@ -57,3 +61,40 @@ server-1.global  10.0.0.8   4648  alive   true    2         3             1.3.0
...
+  ```
+
+  Or use the `-t` flag to format and display the server members information using a Go template:
```
Comment (jrasell) [inline website/content/docs/commands/server/members.mdx:95]: tiny whitespace change:
```suggestion
Or use the `-t` flag to format and display the server members information using a Go template:
```

**Review by jrasell — COMMENTED** [review]
Looking good, thanks!

The PR will need a changelog entry added. Also if we could move the additional tests added to use must, that would be great. An optional change would be to move all the new and current tests in the file to use must, but this isn't required on this PR.

**Review by jrasell — APPROVED** [review]
LGTM, thanks @Juanadelacuesta! Lets add the backport 1.5.x label before merging this in.

File: command/server_members.go [hunk 3]
```diff
@@ -103,6 +115,17 @@ func (c *ServerMembersCommand) Run(args []string) int {
 	// Sort the members
 	sort.Sort(api.AgentMembersNameSort(srvMembers.Members))
 
+	if json || len(tmpl) > 0 {
+		out, err := Format(json, tmpl, srvMembers.Members)
```
Comment (dttung2905) [inline command/server_members.go:119]: Just one minor comment for me. Do we plan to expose `Leader` information when using `-json` flag ?

**Pull request merged**